src/ex4/dist/clients/item_client.js

```diff
@@ -1 +1,35 @@
-// Create an ItemClient class here. This is what makes requests to your express server (your own custom API!)
\ No newline at end of file
+// Create an ItemClient class here. This is what makes requests to your express server (your own custom API!)
+
+export default class item_client {
+    async getTodos() {
+        const response = await fetch("http://localhost:8000/todos");
+        const data = await response.json();
+        return data;
+    }
+
+    async addTodo(todo) {
+        const response = await fetch("http://localhost:8000/todo", {
```

Comment:
```suggestion
        const response = await fetch("http://localhost:8000/todos", {
```

in CRUD api, it's common to use the same name for all endpoints of the same entity..
so:
GET `todos/` will get all list
POST `todos/` will create a new item
GET `todos/{id}` will get a single item
PUT `todos/{id}` will edit a single item
and so on